frontend/src/features/public-form/components/FormPaymentModal/FormPaymentModal.tsx

```diff
@@ -22,6 +22,12 @@ export const FormPaymentModal = ({
   onClose,
   isSubmitting,
 }: FormPaymentModalProps): JSX.Element => {
+  const closeAndSubmit = (event: MouseEvent<HTMLButtonElement>) => {
+    onClose()
```

Comment:
Would recommend adding some comments to explain the calling of `onClose()` as it is not typical for us to close the modal before submitting.

> We need to dismiss the `FormPaymentModal` to release the scroll lock that affects the captcha

Reply:
Thanks Ken! [2fd6bcd](https://github.com/opengovsg/FormSG/pull/6079/commits/2fd6bcd987ae36915f6b217ba4da5fdc30909c3e)